Network/Converter/ObjectState.cs

```diff
@@ -1,21 +1,30 @@
 ﻿namespace Network.Converter
 {
     /// <summary>
-    /// The possible states of a network object.
+    /// The possible states of a network object before and after deserialisation.
     /// </summary>
     public enum ObjectState : byte
     {
         /// <summary>
-        /// The object is null.
-        /// We didn't write something on the stream.
-        /// So we cant read anything from the stream.
+        /// The network object is null, so there is nothing to read from the
+        /// network stream.
         /// </summary>
-        NULL = 0x00,
+        Null = 0x00,
+
         /// <summary>
-        /// The object is not null.
-        /// We wrote something on the stream.
-        /// So we can read something from the stream.
+        /// Identical to <see cref="Null"/>.
         /// </summary>
-        NOT_NULL = 0xFF
+        NULL = Null,
+
+        /// <summary>
+        /// The network object is not null, so there is something to read from
+        /// the network stream.
+        /// </summary>
+        NotNull = 0xFF,
```

Comment:
ObjectState is public. Don't change the enum.

Reply:
this has been rectified, with the old values simply pointing to the new ones. Thus, identical behaviour and compatibility with old code.